src/AddPokemon/AddPokemon.js

```diff
@@ -1,46 +1,35 @@
 import React from "react";
 
 export default function AddPokemon(props) {
-    const { setPokemonCardData, pokemonCardSets, pokemonTypes, save } = props;
+    const {
+        setPokemonCardData,
```

Comment:
it looks like you don't need this prop anymore now that you are using updateCardDetails